prac_09/unreliable_car.py

```diff
@@ -7,20 +7,23 @@
 
 """
 import random
```

Comment:
Maybe only import the module/method you intend on using instead of the whole package.